src/MaterialDesignControls/ControlsMaterial3/IBaseMaterialFieldControl.cs

```diff
@@ -21,5 +21,7 @@ public interface IBaseMaterialFieldControl
         public void SetPlaceholderColor(Color placeHolderColor);
 
         public void SetHorizontalTextAlignment(TextAlignment horizontalTextAlignment);
+
+        public bool ValidateIfAnimate();
```

Comment:
We have to validate together if it will allow us to cover our use cases. I'm not completely sure if we are covering different cases than an empty value (e.g. animate if the Text doesn't have a specific format). With our current approach in V2 we have covered these cases, maybe we should re-think how we can improve it.